package.json

```diff
@@ -96,6 +95,10 @@
     "ts-sinon": "^2.0.2",
     "typescript": "^4.7.3"
   },
+  "resolutions": {
+    "gc-stats/nan": "2.18.0",
+    "express-prometheus-middleware/prometheus-gc-stats/gc-stats": "1.4.1"
+  },
```

Comment:
It was required to compile it in the newer v8 from nodejs 20.

We need to move on from prometheus-gc-stats as it's not maintained since 3 years ago.

Reply:
this isn't a package - but should we but the required node version in the package.json?

```json
  "engines": {
    "node": ">=20"
  },
```